rfcs/2025-carousel.md

```diff
@@ -0,0 +1,480 @@
+</div>
+
+With the **tabbed** style covered, what remains is how to meaningfully convey a **presentational** style carousel to screen readers. As a reminder, the **presentational** is used when rendering a `ListBox` or `GridList` instead of a `SlideShow`, resulting in multiple slides being fully perceivable and interactive at the same time. To detect usage of this style in our hook layer, we can leverage our existing slot utilities.
+
+When in this mode, the implementation strategy will delegate announcements and focus management to the underlying (selectable) collection, leaving rotation controls only responsible for updating the scroll offset. ARIA labels are adapted to convey changes in scroll position instead of perception (e.g. slide visibility), and the `aria-controls` attribute is pointed to the collection as a whole rather than the active slide. 
+
+On wheel events, the `focusedKey` of the underlying collection remains static. Optionally, we may elect to synchronize `focusedKey` when rotation controls are engaged with, allowing the focus restoration logic of the collection to take over when the relationship is followed. 
+
+All remaining accessibility success criteria, such as the ones for automatic rotation, remains in parity with the **tabbed** implementation strategy and in accordance with the [ARIA Carousel Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/). To help digest these differences, the following diagram will present the ARIA semantics of a **presentational** carousel, illustrated in its **multi-view transition** animation model.
+
+<div style="width: 100%; background: #F5F6FC; display: flex; justify-content: center; align-items: center; padding: 2rem 0px 1rem 0px; position: relative; margin-bottom: 12px;">
+  <img src="https://imagedelivery.net/yBU8mJlXW3rs0IX3KL3zRA/6e2fa095-8a91-4111-3eb6-5bde76475a00/Markdown" alt="Presentational Multi-View Carousel Accessibility Anatomy" />
+  <a href="https://imagedelivery.net/yBU8mJlXW3rs0IX3KL3zRA/6e2fa095-8a91-4111-3eb6-5bde76475a00/Markdown" style="position: absolute; bottom: 10px; left: 10px; background: rgba(0,0,0,0.7); color: white; padding: 4px 8px; border-radius: 4px; text-decoration: none; font-size: 12px;">⛶ Fullscreen</a>
+</div>
+
+### Infinite Layouts
+
+For our final design section, we turn to the most ambitious feature of a native carousel implementation: **Infinite mode**. This mode, commonly seen on streaming platforms and hero banners, is used to continuously showcase new or dynamic content and primarily ships in two layout styles:
+
+- **Unordered** - An endlessly revolving list of slides, where the user can scroll seamlessly and the collection appears to wrap at its respective boundaries.
```

Comment:
just to clarify, is this describing infinite scroll + wrapping where hitting the end of your data will loop back to the first slide? If yes, then I'm not quite sure the difference between this and "hierarchical".